src/externalized/DefaultContentManager.cc

```diff
@@ -1163,6 +1164,25 @@ bool DefaultContentManager::loadMercsData()
 	}
 	MERCListingModel::validateData(m_MERCListings);
 
+	std::vector<MERCPROFILESTRUCT*> temp_mercStructs;
+	temp_mercStructs.resize(NUM_PROFILES);
```

Comment:
You can combine these two lines: `std::vector<MERCPROFILESTRUCT*> temp_mercStructs(NUM_PROFILES);
`